lighthouse-core/test/config/budgets-test.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Budgets = require('../../config/budgets');
```

Comment:
```suggestion
const Budgets = require('../../config/budgets.js');
```
(we're slowly trying to go with the fully identified filenames since that's the future)